backend/doc/setting.md

```diff
@@ -504,6 +504,11 @@ For instance, `CODE` edits `Libre Office` files, so extensions will be `odt`, `o
 
 After all these changes in the configuration, you should restart all  process (web, webdav, etc...).
 
+## User Custom Properties
+
+It's possible to configure custom properties attached to user per instance, [read specific documentation
+to know more about this feature]("./user_custom_properties.md").
+
```

Comment:
In development.ini we seen `See the Tracim settings documentation (available in source code, see "settings.md") for more` and here we seen `[read specific documentation
to know more about this feature]("./user_custom_properties.md")`.

- Why not giving directly the good documentation `user_custom_properties.md`?

Reply:
You're right.

Reply:
done.